demos/general/demo_pipeline_voltage_imaging.py

```diff
@@ -105,55 +105,75 @@ def main():
     # first we create a motion correction object with the specified parameters
     mc = MotionCorrect(fnames, dview=dview, **opts.get_group('motion'))
     # Run correction
-    mc.motion_correct(save_movie=True)
+    do_motion_correction = True
+    if do_motion_correction:
```

Comment:
Feels weird to set it and then test it immediately. Why is this a conditional?

Reply:
So here I tried to provide option for users to load the saved motion corrected file and memory mapping file.